examples/objective-c/hello_world/ViewController.m

```diff
@@ -46,8 +47,10 @@ - (void)startEnvoy {
     NSLog(@"starting Envoy failed: %@", error);
   } else {
     NSLog(@"started Envoy, beginning requests...");
-    self.client = [engine streamClient];
+    self.streamClient = [engine streamClient];
+    self.statsClient = [engine statsClient];
     [self startRequests];
+    [self sendStats];
```

Comment:
I just noticed this - it looks like iOS is sending stats only once whereas Android is sending them with each request, is that correct? If so, these should be consistent

Reply:
looking, the structure of the code for ios and android are pretty different